pkg/apis/acid.zalan.do/v1/register.go

```diff
@@ -10,7 +10,8 @@ import (
 
 // APIVersion of the `postgresql` and `operator` CRDs
 const (
-	APIVersion = "v1"
+	APIVersion     = "v1"
+	PostgresqlKind = "postgresql"
```

Comment:
don't get this change if it's only used in one place. I prefer using a const though but when we should do the same with `postgresqlList` and `OperatorConfiguration`.

Reply:
This is a leftover from the implementation where I wanted to attach new objects to `postgresql` instead of statefulset. Indeed, we can probably just get rid of this.